.circleci/run_on_circle.js

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env node
```

Comment:
@makarandp0 Can the filename be something like: `runbuild.js`? We normally don't use `_`s in our filenames.